Compilation error during `docker build`

`docker build` fails with an error during the compilation of `text.cpp`:

```
$ docker build --force-rm --no-cache -t tippecanoe:local .
[+] Building 56.2s (10/10) FINISHED
 => [internal] load build definition from Dockerfile                                                                                                                                                       0.1s
 => => transferring dockerfile: 38B                                                                                                                                                                        0.1s
 => [internal] load .dockerignore                                                                                                                                                                          0.1s
 => => transferring context: 34B                                                                                                                                                                           0.1s
 => [internal] load metadata for docker.io/library/ubuntu:16.04                                                                                                                                            1.5s
 => [internal] load build context                                                                                                                                                                          2.3s
 => => transferring context: 27.93kB                                                                                                                                                                       2.3s
 => CACHED [1/6] FROM docker.io/library/ubuntu:16.04@sha256:1f1a2d56de1d604801a9671f301190704c25d604a416f59e03c04f5c6ffee0d6                                                                               0.0s
 => => resolve docker.io/library/ubuntu:16.04@sha256:1f1a2d56de1d604801a9671f301190704c25d604a416f59e03c04f5c6ffee0d6                                                                                      0.0s
 => [2/6] RUN apt-get update   && apt-get -y upgrade   && apt-get -y install build-essential libsqlite3-dev zlib1g-dev                                                                                    28.7s
 => [3/6] RUN mkdir -p /tmp/tippecanoe-src                                                                                                                                                                 0.5s
 => [4/6] WORKDIR /tmp/tippecanoe-src                                                                                                                                                                      0.0s
 => [5/6] COPY . /tmp/tippecanoe-src                                                                                                                                                                       0.2s
 => ERROR [6/6] RUN make   && make install                                                                                                                                                                25.0s
------
 > [6/6] RUN make   && make install:
#10 0.411 g++ -MMD  -I/usr/local/include -I. -g -Wall -Wshadow -Wsign-compare -Wextra -Wunreachable-code -Wuninitialized -Wshadow -O3 -DNDEBUG  -std=c++11 -c -o geojson.o geojson.cpp
#10 1.517 cc -MMD  -I/usr/local/include -I. -g -Wall -Wshadow -Wsign-compare -Wextra -Wunreachable-code -Wuninitialized -Wshadow -O3 -DNDEBUG  -c -o jsonpull/jsonpull.o jsonpull/jsonpull.c
#10 1.835 g++ -MMD  -I/usr/local/include -I. -g -Wall -Wshadow -Wsign-compare -Wextra -Wunreachable-code -Wuninitialized -Wshadow -O3 -DNDEBUG  -std=c++11 -c -o tile.o tile.cpp
#10 7.863 g++ -MMD  -I/usr/local/include -I. -g -Wall -Wshadow -Wsign-compare -Wextra -Wunreachable-code -Wuninitialized -Wshadow -O3 -DNDEBUG  -std=c++11 -c -o pool.o pool.cpp
#10 7.935 g++ -MMD  -I/usr/local/include -I. -g -Wall -Wshadow -Wsign-compare -Wextra -Wunreachable-code -Wuninitialized -Wshadow -O3 -DNDEBUG  -std=c++11 -c -o mbtiles.o mbtiles.cpp
#10 9.826 g++ -MMD  -I/usr/local/include -I. -g -Wall -Wshadow -Wsign-compare -Wextra -Wunreachable-code -Wuninitialized -Wshadow -O3 -DNDEBUG  -std=c++11 -c -o geometry.o geometry.cpp
#10 17.23 g++ -MMD  -I/usr/local/include -I. -g -Wall -Wshadow -Wsign-compare -Wextra -Wunreachable-code -Wuninitialized -Wshadow -O3 -DNDEBUG  -std=c++11 -c -o projection.o projection.cpp
#10 17.34 g++ -MMD  -I/usr/local/include -I. -g -Wall -Wshadow -Wsign-compare -Wextra -Wunreachable-code -Wuninitialized -Wshadow -O3 -DNDEBUG  -std=c++11 -c -o memfile.o memfile.cpp
#10 17.39 g++ -MMD  -I/usr/local/include -I. -g -Wall -Wshadow -Wsign-compare -Wextra -Wunreachable-code -Wuninitialized -Wshadow -O3 -DNDEBUG  -std=c++11 -c -o mvt.o mvt.cpp
#10 20.03 g++ -MMD  -I/usr/local/include -I. -g -Wall -Wshadow -Wsign-compare -Wextra -Wunreachable-code -Wuninitialized -Wshadow -O3 -DNDEBUG  -std=c++11 -c -o serial.o serial.cpp
#10 21.68 g++ -MMD  -I/usr/local/include -I. -g -Wall -Wshadow -Wsign-compare -Wextra -Wunreachable-code -Wuninitialized -Wshadow -O3 -DNDEBUG  -std=c++11 -c -o main.o main.cpp
#10 24.85 g++ -MMD  -I/usr/local/include -I. -g -Wall -Wshadow -Wsign-compare -Wextra -Wunreachable-code -Wuninitialized -Wshadow -O3 -DNDEBUG  -std=c++11 -c -o text.o text.cpp
#10 24.94 text.cpp: In function 'int integer_zoom(std::__cxx11::string, std::__cxx11::string)':
#10 24.94 text.cpp:134:17: error: 'isfinite' was not declared in this scope
#10 24.94   if (!isfinite(d) || d != floor(d) || d < 0 || d > 32) {
#10 24.94                  ^
#10 24.94 text.cpp:134:17: note: suggested alternative:
#10 24.94 In file included from milo/dtoa_milo.h:4:0,
#10 24.94                  from text.cpp:6:
#10 24.94 /usr/include/c++/5/cmath:601:5: note:   'std::isfinite'
#10 24.94      isfinite(_Tp __x)
#10 24.94      ^
#10 24.96 Makefile:74: recipe for target 'text.o' failed
#10 24.96 make: *** [text.o] Error 1
------
executor failed running [/bin/sh -c make   && make install]: exit code: 2
```

--- a/text.cpp
+++ b/text.cpp
@@ -131,7 +131,7 @@ std::string truncate16(std::string const &s, size_t runes) {
 
 int integer_zoom(std::string where, std::string text) {
 	double d = atof(text.c_str());
-	if (!isfinite(d) || d != floor(d) || d < 0 || d > 32) {
+	if (!std::isfinite(d) || d != floor(d) || d < 0 || d > 32) {
 		fprintf(stderr, "%s: Expected integer zoom level in \"tippecanoe\" GeoJSON extension, not %s\n", where.c_str(), text.c_str());
 		exit(EXIT_JSON);
 	}

